### PR TITLE
Add SVGcode article

### PR DESCRIPTION
**Pull request by tomayac**
Changes proposed in this pull request:

- Add the article.

**Merge this only _after_ the video has been released.**

Fixes https://github.com/GoogleChrome/web.dev/issues/6833.

### CODE REVIEW

File: src/site/content/en/blog/svgcode/index.md [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+
+#### Window Controls Overlay
+
+To maximize the available screen real estate, SVGcode makes use of
+[Window Controls Overlay](/window-controls-overlay/) customization by moving its main menu up into
+the titlebar area. You can see this get activated at the end of the install flow.
+
+{% Video autoplay=true,
+              muted=true,
+              loop=true,
```
[the remark this answers is not in the document]
Comment (jpmedley) [reply src/site/content/en/blog/svgcode/index.md:109]: Please do likewise with the rest of your video "images".

File: src/site/content/en/blog/svgcode/index.md [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+SVGcode is also fully integrated with the operating system's clipboard via the Async Clipboard API.
+You can paste images from the operating system's file explorer into the app either by clicking the
+paste image button or by pressing command or control plus v on your keyboard.
+
+{% Video autoplay=true,
+              muted=true,
+              loop=true,
+              playsinline=true,  src="video/8WbTDNrhLsU0El80frMBGE4eMCD3/9wHL0Uc7eHFEFF99anaB.mp4" %}
+
+The Async Clipboard API recently has gained the ability to deal with SVG images as well, so you can
```
Comment (jpmedley) [inline src/site/content/en/blog/svgcode/index.md:193]:
```suggestion
The Async Clipboard API has recently gained the ability to deal with SVG images as well, so you can
```

File: src/site/content/en/blog/svgcode/index.md [hunk 3]
```diff
@@ -0,0 +1,270 @@
...
+      );
+      inputImage.src = blobURL;
+      await set(FILE_HANDLE, handle);
+      return;
+    }
+  }
+});
+```
+
+Read up all the details on [file handling](/file-handling/) in article, and view the source code in
```
Comment (jpmedley) [inline src/site/content/en/blog/svgcode/index.md:258]:
```suggestion
For more information, see [Let installed web applications be file handlers](/file-handling/), and view the source code in
```

File: src/site/content/en/blog/svgcode/index.md [hunk 4]
```diff
@@ -0,0 +1,270 @@
...
+[`src/js/filehandling.js`](https://github.com/tomayac/SVGcode/blob/main/src/js/filehandling.js).
+
+## Conclusion
+
+Alright, this was a quick tour through some of the advanced app features in SVGcode. I hope this app
+can become an essential tool for your image processing needs, alongside other amazing apps like
```
Comment (jpmedley) [inline src/site/content/en/blog/svgcode/index.md:264]:
```suggestion
can become an essential tool for your image processing needs alongside other amazing apps like
```

File: src/site/content/en/blog/svgcode/index.md [hunk 5]
```diff
@@ -0,0 +1,270 @@
...
+[Squoosh](https://squoosh.app/) or [SVGOMG](https://jakearchibald.github.io/svgomg/).
+
+SVGcode is available at [svgco.de](https://svgco.de/), see what I did there? You can
```
Comment (jpmedley) [inline src/site/content/en/blog/svgcode/index.md:267]:
```suggestion
SVGcode is available at [svgco.de](https://svgco.de/). See what I did there? You can
```

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: src/site/content/en/blog/svgcode/index.md [hunk 6]
```diff
@@ -0,0 +1,270 @@
...
+{% YouTube "kcvfyQh6J-0" %}
+
+## From raster to vector
+
+Have you ever scaled up or down an image and the result was pixelated and not very satisfactory? If
+so, you have probably dealt with a raster image format such as WebP, PNG, or JPG.
+
+{% Video autoplay=true,
+              muted=true,
+              loop=true,
```
[the remark this answers is not in the document]
Comment (tomayac) [reply src/site/content/en/blog/svgcode/index.md:34]: It looks like an image, but it's technically still a `<video>`, and there is no `alt` attribute allowed on those. Instead, let me wrap all `{% Video %}` in `<figure>`.

File: src/site/content/en/blog/svgcode/index.md [hunk 7]
```diff
@@ -0,0 +1,270 @@
...
+              playsinline=true,  src="video/8WbTDNrhLsU0El80frMBGE4eMCD3/bIiC6vyZLqgGWPuFF9od.mp4" %}
+
+In contrast, vector graphics are images that are defined by points in a coordinate system. These
+points are connected by lines and curves to form polygons and other shapes. Vector graphics have the
+unique advantage over raster graphics in that they may be scaled up or down to any resolution
+without getting pixelated.
+
+{% Video autoplay=true,
+              muted=true,
+              loop=true,
```
[the remark this answers is not in the document]
Comment (tomayac) [reply src/site/content/en/blog/svgcode/index.md:44]: See above

File: src/site/content/en/blog/svgcode/index.md [hunk 8]
```diff
@@ -0,0 +1,270 @@
...
+              playsinline=true,  src="video/8WbTDNrhLsU0El80frMBGE4eMCD3/yM32DfKFp8ooBAshjlUE.mp4" %}
+
+## Introducing SVGcode
+
+I have built a PWA called [SVGcode](https://svgco.de/) that can help you convert raster images to
+vectors. Credit where credit is due: I didn't invent this. With SVGcode, I just stand on the
+shoulders of a command line tool called [Potrace](http://potrace.sourceforge.net/) by
+[Peter Selinger](https://www.mathstat.dal.ca/~selinger/) that I have
+[converted to Web Assembly](https://www.npmjs.com/package/esm-potrace-wasm), so it can be used in a
```
[the remark this answers is not in the document]
Comment (tomayac) [reply src/site/content/en/blog/svgcode/index.md:53]: This required that I knew what I was doing, though. Maybe something for @RReverser?!

File: src/site/content/en/blog/svgcode/index.md [hunk 9]
```diff
@@ -0,0 +1,270 @@
...
+that I have downloaded from the ChromiumDev Twitter channel. This is a PNG raster image that I then
+drag onto the SVGcode app. When I drop the file, the app starts tracing the image color by color,
+until a vectorized version of the input appears. I can now zoom into the image, and as you can see,
+the edges stay sharp. But zooming in on the Chrome logo, you can see that the tracing wasn't
+perfect, and especially the outlines of the logo look a bit speckled. I can improve the result by
+de-speckling the tracing by suppressing speckles of up to, say, 5 pixels.
+
+{% Video autoplay=true,
+              muted=true,
+              loop=true,
```
[the remark this answers is not in the document]
Comment (tomayac) [reply src/site/content/en/blog/svgcode/index.md:70]: See above.

File: src/site/content/en/blog/svgcode/index.md [hunk 10]
```diff
@@ -0,0 +1,270 @@
...
+### Posterization in SVGcode
+
+An important step for vectorization, especially for photographic images, is posterizing the input
+image to reduce the number of colors. SVGcode allows me to do this per color channel, and see the
+resulting SVG as I make changes. When I'm happy with the result, I can save the SVG to my harddisk
+and use it wherever I like.
+
+{% Video autoplay=true,
+              muted=true,
+              loop=true,
```
[the remark this answers is not in the document]
Comment (tomayac) [reply src/site/content/en/blog/svgcode/index.md:82]: See above.

File: src/site/content/en/blog/svgcode/index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (tomayac) [reply src/site/content/en/blog/svgcode/index.md:109]: See above.

**Comment by tomayac** [comment]
Thanks, @jpmedley, for the review! PTAL!

**Review by jpmedley — APPROVED** [review]

**Pull request merged**